support-e2e/tests/test/checkout.ts

```diff
@@ -1,17 +1,52 @@
-import { expect, test } from '@playwright/test';
+import { expect, Page, test } from '@playwright/test';
 import { email, firstName, lastName } from '../utils/users';
 import { setupPage } from '../utils/page';
-import { setTestUserRequiredDetails } from '../utils/testUserDetails';
+import {
+	setTestUserDetails,
+	setTestUserRequiredDetails,
+} from '../utils/testUserDetails';
 import { fillInPayPalDetails } from '../utils/paypal';
 import { fillInCardDetails } from '../utils/cardDetails';
 import { checkRecaptcha } from '../utils/recaptcha';
+import { TestFields, ukWithPostalAddressOnly } from '../utils/userFields';
 
+// TODO: it'd be great to make the types here more specific, possibly using the
+// shared types from the product catalog.
 type TestDetails = {
 	product: string;
 	ratePlan: string;
 	paymentType: string;
 	internationalisationId: string;
-	paymentFrequency: string;
+};
+
+const setUserDetailsForProduct = async (
+	page,
+	product,
+	internationalisationId,
+) => {
+	switch (product) {
+		case 'SupporterPlus':
```

Comment:
In a seperate PR, maybe we could rename to `All-Access Digital` and maybe introduce a Tier1 `Support` test?